README.md

```diff
@@ -1,18 +1,692 @@
+- Incorrect: "Installing Temporal"
+
+#### Infinitive verb forms in labels
+
+Treat labels like headings or titles and use infinitive verb forms when possible.
+
+- Correct: "Install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Sentence casing in headings
+
+Use sentence casing for titles and headings.
+Sentence casing means that only proper nouns and the first letter of the first word are capitalized.
+
+- Correct: "How to get started with Temporal"
+- Incorrect: "How To Get Started With Temporal"
+
+### What is the philosophy around versioning the documentation?
+
+The Temporal Platform includes many different components and core dependencies. Many components are independently versioned, meaning that they document their stability and support for their own dependencies. The [Temporal Go SDK reference](https://pkg.go.dev/go.temporal.io/sdk?tab=versions) provides a good example of document versioning.
+
+The goal of this information set, in regards to versioning, is to remain “current”. That is, this information should serve the needs of the Platform’s user base as best it can based on what is/has recently happened across all the components of the Temporal Platform.
+Make explicit callouts to support, stability, and dependency information whenever possible.
+
+We use the `ssdi` metadata tag in `docs-src` nodes to explicitly call out this information.
+
+#### How to explicitly identify support, stability, and dependency info
+
+Use the `ssdi` metadata in a `docs-src` node to explicitly call out support, stability, and dependency information.
+Example:
+
+```markdown
+---
+id: what-is-x
+// ...
+ssdi:
+  - Introduced in [Temporal Server version x.xx.x](/path/to/release/notes/)
+  - Available in [Temporal CLI version x.xx.x](/path/to/release/notes/)
+  - Released in Temporal Cloud on <date>
+---
+```
+
+### How to auto format files
+
+Run `yarn format`.
+
+If you don't run this command locally before posting a Pull Request, a Github action runs the formatting.
+
+### How to run the Assembly Workflow
+
+Make sure you have a Namespace set up to operate out of.
+We recommend using the [Temporal CLI’s development Server](https://docs.temporal.io/cli/#starting-the-temporal-server).
+
+In a separate terminal run:
+
+```bash
+temporal server start-dev
+```
+
+In a separate terminal, run the Worker in the `/assembly` folder.
+
+```bash
+cd assembly
+./worker.js
+```
+
+In another terminal, start the Workflow from the root of this repository.
+
+```bash
+yarn assemble
+```
+
+Run the auto-formatter after the Workflow Execution is completed.
+
+```bash
+yarn format
+```
+
+**Make your changes in the *source files*  named in the configuration file.**
+For instance, if you find a typo under "What is a Task?" (located in the Workers section of the Temporal docs website), open [docs/concepts/what-is-a-task.md](https://github.com/temporalio/documentation/blob/main/docs/concepts/what-is-a-task.md) and make the edit directly there.
```

Comment:
This paragraph seems oddly placed. Perhaps move the example up into the "How to fix a typo" section?